setup.cfg

```diff
@@ -39,21 +39,29 @@ install_requires =
 cibuildwheel = resources/*
 
 [options.extras_require]
-dev =
-    click
+docs =
     mkdocs-include-markdown-plugin==2.1.1
     mkdocs==1.0.4
+    pymdown-extensions
+test =
+    jinja2
+    pytest
+    pytest-timeout
+dev =
+    click
+    ghapi
+    jinja2
+    mypy>=0.800
+    packaging>=20.8
```

Comment:
Looks like we lost pytest from `dev`. Should we add the `test` extra to the requirements-dev? or add pytest in twice?

Maybe the best solution would be to define these in setup.py, after all. then `requirements-dev` can match `dev` and `dev can contain everything.

Reply:
Okay, moved them over, also added an "all" which installs everything. I'd rather keep the docs requirements out of dev, since I don't use the same venv for dev and docs; it's easy to specify two extras (or `all` now).